pkg/apis/channels/v1alpha1/subscription_validation_test.go

```diff
@@ -30,34 +30,48 @@ func TestSubscriptionSpecValidation(t *testing.T) {
 	}{{
 		name: "valid",
 		c: &SubscriptionSpec{
-			Channel:    "bar",
-			Subscriber: "foo",
+			From:      "fromChannel",
+			Processor: "processor",
 		},
 		want: nil,
 	}, {
 		name: "valid with arguments",
 		c: &SubscriptionSpec{
-			Channel:    "bar",
-			Subscriber: "foo",
-			Arguments:  &[]Argument{{Name: "foo", Value: "bar"}},
+			From:      "fromChannel",
+			Processor: "processor",
+			Arguments: &[]Argument{{Name: "foo", Value: "bar"}},
 		},
 		want: nil,
 	}, {
-		name: "missing subscriber",
+		name: "missing processor and to",
 		c: &SubscriptionSpec{
-			Channel: "foo",
+			From: "fromChannel",
 		},
 		want: func() *apis.FieldError {
-			fe := apis.ErrMissingField("subscriber")
-			fe.Details = "the Subscription must reference a Subscriber"
+			fe := apis.ErrMissingField("to", "processor")
+			fe.Details = "the Subscription must reference a to channel or a processor"
 			return fe
 		}(),
+	}, {
+		name: "missing to",
+		c: &SubscriptionSpec{
+			From: "fromChannel",
+			To:   "toChannel",
```

Comment:
Should this be "processor", or should "to" be left off? (This matches the next test case right now.)